augeventhandlers.js

```diff
@@ -0,0 +1,73 @@
+//Event Handlers IIFE
+// THIS IIFE IS FOR EVENTLISTENERS
+var CarLot = (function(carlot) {
+
+  // DOM element variables
+  var cardsInDOM = document.getElementsByClassName("cards"); // cardsInDOM is an array of inventory divs, class given to div that holds car elements
+  var form = document.getElementsByClassName("form");
```

Comment:
With only one form on the page, and moreover, only using a single element out of this array of DOM elements, I'm not sure why you didn't just give the form an ID and select by that.